2021/_data/talks.json

```diff
@@ -199,6 +221,25 @@
         "443a683fa84a77151521434bf96a7a082d29d361"
       ]
     },
+    {
+      "id": "60d0df511fc64e0884ea9eeb",
+      "title": "Orchestrating Robot Swarms with Java",
+      "languages": [
+        "eng"
+      ],
+      "abstract": "At Ocado Technology, we are heavily invested in Java. We are using this language in our in-house built, end-to-end platform Ocado Smart Platform (OSP), from ecommerce to bots fulfilling customer orders and routing deliveries to homes. OSP is the world’s most advanced end-to-end eCommerce, logistics and fulfilment platform ever developed for online grocery, used by some of the world’s most forward-thinking grocery retailers across the globe.\n\nOur highly automated fulfilment centre enlist thousands of bots to process hundreds of thousands of orders per week (click here if you would like to see our bots in action).\n\nIn this session, we will uncover Java’s role in orchestrating these huge bot swarms for superior efficiencies of scale and explore some of the benefits and challenges that the use of Java has presented. A common question we get asked (and ask ourselves!) is why Java instead of others, like C++ or Rust, for example. Over the next 50 minutes, we will explore the four main reasons behind this decision: \n\n- Performance\n- Speed of development\n- Evolving platform\n- Recruitment\n\nJava helps us to not only optimize our control system but also the productivity of our developers. Throughout this presentation, some exciting topics that we will dig into are: \n\n- The different decisions that have to be made at the beginning of an application’s life and the choices we have made at Ocado Technology in developing our robotic swarm control system\n- The use of Java in a scenario where other lower-level languages might seem more appropriate\n- Development principles we adhere to and the implications they have on the implementation choices within our system",
```

Comment:
```suggestion
      "abstract": "At Ocado Technology, we are heavily invested in Java. We are using this language in our in-house built, end-to-end platform Ocado Smart Platform (OSP), from ecommerce to bots fulfilling customer orders and routing deliveries to homes. OSP is the world’s most advanced end-to-end eCommerce, logistics and fulfilment platform ever developed for online grocery, used by some of the world’s most forward-thinking grocery retailers across the globe.\n\nOur highly automated fulfilment centre enlist thousands of bots to process hundreds of thousands of orders per week.\n\nIn this session, we will uncover Java’s role in orchestrating these huge bot swarms for superior efficiencies of scale and explore some of the benefits and challenges that the use of Java has presented. A common question we get asked (and ask ourselves!) is why Java instead of others, like C++ or Rust, for example. In this talk, we will explore the four main reasons behind this decision: \n\n- Performance\n- Speed of development\n- Evolving platform\n- Recruitment",
```